For Ember Times No. 112: Ember Octane Release Plan Writeup (#246)
ember-learn/ember-blog: source/2019-08-23-the-ember-times-issue-112.md

Gi suilon Emberistas! 🐹

This week:
watch talks from 1st Chennai EmberJS Meetup 🇮🇳,
announcing the official Ember Octane release plan 🏎🐹,
<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>

---

## [Ember Octane Is Coming 🏎🐹](https://blog.emberjs.com/2019/08/15/octane-release-plan.html)

The time has come: [Ember Octane](https://emberjs.com/editions/) is coming to your Ember application very soon!
As [recently announced in the official Ember Octane release plan](https://blog.emberjs.com/2019/08/15/octane-release-plan.html), the very first edition of Ember is going to be released at the same time as the planned release date of Ember v3.14.

> But what does it mean? By the time Ember 3.14 is published, the framework will include all Octane-related features and enable them by default. Additionally, the "Octane way" will be the primary and recommended way to build Ember apps. This will also be reflected in Ember's official documentation (API Docs and Guides) and migration tools.

Author: ```suggestion
But what does it mean? By the time Ember 3.14 is published, the framework will include all Octane-related features and enable them by default. Additionally, the "Octane way" will be the primary and recommended way to build Ember apps. This will also be reflected in Ember's official documentation ([API Docs](https://api.emberjs.com/ember/release) and [Guides](https://octane-guides-preview.emberjs.com/release/upgrading/editions/)) and migration tools.
```

> But what does it mean? By the time Ember 3.14 is published, the framework will include all Octane related features and these will be enabled by default, too. Additionally, the "Octane way" will be the primary and recommended way to build Ember apps. This will also be reflected in Ember's official documentation, like the API Docs or the Guides, and migration tools.

Reviewer: In addition to the suggestion below, what do you think about adding links to API Docs and Guides? I know there's the preview Octane Guides, but wasn't sure if the URL would be back to blog.emberjs.com once Octane is released.
```suggestion
But what does it mean? By the time Ember 3.14 is published, the framework will include all Octane-related features and enable them by default. Additionally, the "Octane way" will be the primary and recommended way to build Ember apps. This will also be reflected in Ember's official documentation (API Docs and Guides) and migration tools.
```

> Wanna learn more about what is going to happen between now and the time Ember Octane lands? Looking for support on migrating and testing your apps with the Octane feature set? Be sure to give the [official release plan a read](https://blog.emberjs.com/2019/08/15/octane-release-plan.html) and **get ready for Octane**!

Reviewer: I think you could also mention and link to the `#topic-octane-migration` channel on Discord. 

I'm having difficulty finding a direct link to the channel for you (not the link to the Discord invite). Amy found one for me, and I'm not sure how. :(
```suggestion
Wanna learn more about what is going to happen between now and the time Ember Octane lands? Looking for support for migrating and testing your apps with the Octane feature set? Be sure to give the [official release plan a read](https://blog.emberjs.com/2019/08/15/octane-release-plan.html) and **get ready for Octane**!
```

> Wanna learn more about what is going to happen between now and the time Ember Octane lands? Looking for support for migrating and testing your apps with the Octane feature set? Be sure to give the [official release plan a read](https://blog.emberjs.com/2019/08/15/octane-release-plan.html) and **get ready for Octane**!

Author: ```suggestion
Wanna learn more about what is going to happen between now and the time Ember Octane lands? Looking for support for migrating and testing your apps with the Octane feature set? Be sure to give the [official release plan a read](https://blog.emberjs.com/2019/08/15/octane-release-plan.html), join the discussion on the [#topic-octane-migration](https://discordapp.com/channels/480462759797063690/608346628163633192) channel on the [Ember Discord](https://discordapp.com/invite/zT3asNS) and **get ready for Octane**!
```
